package.json

```diff
@@ -47,6 +47,11 @@
     "> 0.25%, not dead"
   ],
   "dependencies": {
+    "@aws-sdk/client-s3": "3.535.0",
+    "@aws-sdk/lib-storage": "3.535.0",
+    "@aws-sdk/s3-presigned-post": "3.535.0",
```

Comment:
My local env can work without importing as expected. But CI cannot find this module so I imported it here. Don't know the root cause and I found a similar issue with `aws-sdk/middleware-endpoint` before(https://github.com/aws/aws-sdk-js-v3/issues/3983).